mondey_backend/src/mondey_backend/users.py

```diff
@@ -18,29 +18,30 @@
 
 from .databases.users import AccessToken
 from .databases.users import User
-from .databases.users import async_session_maker
 from .databases.users import get_access_token_db
 from .databases.users import get_user_db
 from .settings import app_settings
 
 
+def send_email_validation_link(email: str, token: str) -> None:
+    msg = EmailMessage()
+    msg["From"] = "no-reply@mondey.lkeegan.dev"
+    msg["To"] = email
+    msg["Subject"] = "MONDEY-Konto aktivieren"
+    msg.set_content(
+        f"Bitte klicken Sie hier, um Ihr MONDEY-Konto zu aktivieren:\n\nhttps://mondey.lkeegan.dev/verify/{token}"
```

Comment:
should this be localized in some way or contain an english translation?

Reply:
I guess so, I'd go with including an english translation in the email to keep this simple